cpp/server/ct-server.cc

```diff
@@ -132,6 +137,22 @@ Gauge<>* latest_local_tree_size_gauge =
     Gauge<>::New("latest_local_tree_size",
                  "Size of latest locally generated STH.");
 
+Counter<>* sequencer_total_runs =
+    Counter<>::New("sequencer_total_runs", "Total number of sequencer runs.");
+Latency<milliseconds> sequencer_run_latency_ms("sequencer_run_latency_ms",
+                                               "Total runtime of sequencer");
+Latency<milliseconds> sequencer_delete_latency_ms(
+    "sequencer_delete_latency_ms",
+    "Total time spent deleting entries by sequencer");
+Latency<milliseconds> sequencer_sequence_latency_ms(
+    "sequencer_sequence_latency_ms",
+    "Total time spent sequencing entries by sequencer");
+
+Counter<>* signer_total_runs =
+    Counter<>::New("signer_total_runs", "Total number of signer runs.");
```

Comment:
Isn't that included as part of ```signer_run_latency_ms```? But looking below, this is really the number of *successful* runs, could just update the docstring.